### PR TITLE
Remove support for standalone

### PR DESCRIPTION
**Pull request by labkey-adam**
#### Changes
* Remove IntelliJ standalone run configurations
* Remove `/webapps`, including `labkey.xml`
* Begin dismantling `tomcat-lib` handling

### CODE REVIEW

**Review by labkey-jeckels — COMMENTED** [review]
Looks OK to me but I'll defer to Susan for a more informed review on Artifactory usage, etc.

File: server/build.gradle [hunk 1]
```diff
@@ -29,8 +29,6 @@ BuildUtils.addLabKeyDependency(project: project, config: 'tomcatJars', depProjec
 
 dependencies
         {
-            tomcatJars "com.sun.mail:jakarta.mail:${javaMailVersion}"
-            tomcatJars "jakarta.activation:jakarta.activation-api:${activationApiVersion}" // JavaMail dependency, so it must be copied to <tomcat>/lib
```
Comment (labkey-susanh) [inline server/build.gradle:33]: I assume it's on your radar to remove the tomcatJars configuration (in LabKey.groovy in the gradle plugins).

**Review by labkey-susanh — APPROVED** [review]

File: server/build.gradle (hunk 1 above)
Comment (labkey-adam) [reply server/build.gradle:33]: Yes, `tomcatJars`, `tomcatLibDir`, `useEmbeddedTomcat`, et al are all going away

Comment (labkey-adam) [reply server/build.gradle:33]: For now, I'm clearing what I can from the build prior to publishing a new plugin

**Pull request merged**